doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -157,14 +157,20 @@ compatible with the rest of |Coq|, up to a few discrepancies:
   (see :ref:`pattern_conditional_ssr`).  To use the
   generalized form, turn off the |SSR| Boolean ``if`` notation using the command:
   ``Close Scope boolean_if_scope``.
-+ The following two options can be unset to disable the incompatible
-  rewrite syntax and allow reserved identifiers to appear in scripts.
++ The following flags can be unset to make |SSR| more compatible with
+  parts of Coq:
 
-  .. coqtop:: in
+.. flag:: SsrRewrite
+
+   Controls whether the incompatible rewrite syntax is required.  The default
+   is "on".  "off" makes the syntax compatible with other parts of Coq.
+
+   **Reviewers: "required" or "allowed" above?**
```

Comment:
Rather "activated"... ? (thus closer to "allowed" or "supported")

Reply:
I made it "enabled".  Presumably readers will grasp that enabled=set and disabled=unset.